server/monitoring/monitoring.go

```diff
@@ -48,6 +51,7 @@ func Monit() {
 		} else {
 			if found > 0 {
 				logger.Info("monitoring", fmt.Sprintf("Deleted %d entries and files in %s", found, time.Since(now)))
+				statistics.S.UpdateCurrentSize(sizeRemoved)
```

Comment:
Well if I see correctly, you're setting the S.CurrentSize to the size you deleted, and that's not relevent I think. 

Yes you're right about the transaction ^^ My code is really far from being perfect, I appreciate all the help I can get to improve myself :)